scanpy/plotting/_utils.py

```diff
@@ -742,9 +742,7 @@ def scatter_base(
             rectangle = [left, bottom, width, height]
             fig = pl.gcf()
             ax_cb = fig.add_axes(rectangle)
-            cb = pl.colorbar(
-                sct, format=ticker.FuncFormatter(ticks_formatter), cax=ax_cb
-            )
+            pl.colorbar(sct, format=ticker.FuncFormatter(ticks_formatter), cax=ax_cb)
```

Comment:
Why did this line change? I assume black would get this.

Reply:
unused variable

Reply:
Ah, didn't see that

Why is this sometimes changed to `_ = statement` in other places? Was this an automatic tool, or manual?

Reply:
Manual.

_ is the consensus for variables that get thrown away.

__ is not the standard -> I had to rename those as well